modules/core/src/main/java/org/apache/ignite/internal/util/tostring/GridToStringBuilder.java

```diff
@@ -906,9 +918,120 @@ else if (val instanceof Map && ((Map)val).size() > MAX_COL_SIZE) {
 
             // No other option here.
             throw new IgniteException(e);
+        } finally {
+            svdObjects.remove(obj);
+        }
+    }
+
+    /**
+     * Creates an uniformed string presentation for the given object.
+     * We need this method to count every Object to prevent infinite toString() loops.
+     *
+     * @param <T> Type of object.
+     * @param buf String builder.
+     * @param obj Object for which to get string presentation.
+     * @param svdObjects Map with objects already presented in the buffer.
+     */
+    @SuppressWarnings({"unchecked"})
+    private static <T> void toStringAddVal(T obj, SB buf, IdentityHashMap<Object, Integer> svdObjects) {
```

Comment:
Any reason name StringBuilder as "bud"?
Let's named it "sb"?

Reply:
The reason is to have same name for the same object in toStringImpl() and toStringAddVal().